src/test/java/com/wikia/webdriver/common/remote/operations/CreateCategory.java

```diff
@@ -29,11 +32,19 @@
         .put("siteId", context.getSiteId())
         .build());
 
-    final String response = remoteOperation.execute(buildUrl(context), jsonObject);
+    String response = null;
+    try {
+      response = remoteOperation.execute(buildUrl(context), jsonObject);
+    } catch(RemoteException e) {
+      if(e.toString().contains(MAX_CATEGORIES_REACHED)) {
+        // TODO: remove all categories except "General", "All"
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1135)